### PR TITLE
react blog on development branch

### PR DESCRIPTION
**Pull request by frankchoongsaeng**
Hi, I've made some progress on my react blog.
Hope you see what you like

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import Router from './Components/Router';
 
 function App() {
   return (
-    <div>
-      <Article />
-    </div>
+    <ThemeProvider >
+      <Router />
+    </ThemeProvider>
```
Comment (zustmornah) [inline src/App.js:9]: Nice layout for your components !

File: src/Components/ThemeButton.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  function changeTheme() {
+    themeContext[1](themeContext[0] === "light" ? "dark" : "light");
+  }
+
+  return (
+    <button onClick={changeTheme}>Change Theme</button>
+  );
+}
+
+export default ThemeButton;
```
Comment (zustmornah) [inline src/Components/ThemeButton.js:17]: cool addition, dark and light themes!

**Review by zustmornah — APPROVED** [review]
Good push Sir, i really liked how your project structure...

File: src/App.js (hunk 1 above)
Comment (frankchoongsaeng) [reply src/App.js:9]: Thanks Herman.

**Pull request merged**